src/moonlink_connectors/src/pg_replicate.rs

```diff
@@ -10,6 +10,9 @@ pub mod table;
 pub mod table_init;
 pub mod util;
 
+#[cfg(test)]
```

Comment:
nit: we usually put test mod at the end of file, more of a repo-wise coding convention.
example: https://github.com/Mooncake-Labs/moonlink/blob/bf343c637283d6af6fcbbb1375cae00065ebd07e/src/moonlink/src/storage/mooncake_table.rs#L1620-L1651